apps/storybook/stories/components/data-display/card/Card.mdx

```diff
@@ -8,81 +9,108 @@ import {
   CardCenteringString,
   CardHorizontalString,
 } from "./srcStrings.ts";
-import { Card, CardBody } from "@kvib/react/src";
+import { Card, CardBody, Text, Code, Box, Alert, AlertIcon, AlertDescription } from "@kvib/react/src";
 
 <Meta title="Komponenter/Data Display/Card" />
 
 # Card
 
 Card er et fleksibelt komponent som brukes til å gruppere og vise innhold i et klart og konsist format.
 
-## Ta i bruk
-
 ```jsx
 import { Card, CardHeader, CardBody, CardFooter } from "@kvib/react";
 ```
 
-## Props
+<DocsContainer>
 
-<Card variant="outline" borderRadius="0">
-  <CardBody>
-    <Story of={CardStories.Card} />
-  </CardBody>
-</Card>
+**Egnet til:**
 
-<Source code={CardString} dark />
-<Controls of={CardStories.Card} />
+- Gruppere innhold
+- Gruppere innhold og handlinger som hører sammen
+- Inngangspunkt til mer kompleks og detaljert informasjon
+
+**Uegnet til:**
+
+- Kompleks og detaljert informasjon som krever mye plass
 
-## Varianter
+<Feedback component="Card" />
 
-<Card variant="outline" borderRadius="0">
-  <CardBody>
-    <Story of={CardStories.CardVariants} />
-  </CardBody>
-</Card>
+</DocsContainer>
 
-<Source code={CardVariantsString} dark />
+<DocsHeading>Kontekst</DocsHeading>
 
-## Størrelser
+<DocsContainer>
 
-<Card variant="outline" borderRadius="0">
-  <CardBody>
-    <Story of={CardStories.CardSizes} />
-  </CardBody>
-</Card>
+**Card er bygget opp av fire komponenter:**
 
-<Source code={CardSizesString} dark />
+- Card
+- CardHeader
+- CardBody
+- CardFooter.
 
-## Card med bilde og knapper
+`Card` skal alltid være ytterst. `CardBody` skal inneholde hoveddelen av innholdet. `CardHeader` og `CardFooter` er valgfrie.
 
-<Card variant="outline" borderRadius="0">
-  <CardBody>
-    <Story of={CardStories.CardImage} />
-  </CardBody>
-</Card>
+</DocsContainer>
 
-<Source code={CardImageString} dark />
+<DocsHeading>Alternativer</DocsHeading>
 
-## Sentrering av innhold
+<Alert status="info" padding="0.3rem 1rem">
+<AlertIcon />
+<AlertDescription>
+  Kodevisningen viser `<CardHeader2>`/`<CardBody2>`/`<CardFooter2>` - dette skal være `<CardHeader>`/`<CardBody>`/`<CardFooter>` i faktisk bruk av komponentet.
+</AlertDescription>
+</Alert>
 
-Sentrering av innhold gjøres ved å sette `align="center"` på Card.
+<DocsContainer>
 
-<Card variant="outline" borderRadius="0">
-  <CardBody>
-    <Story of={CardStories.CardCentering} />
-  </CardBody>
-</Card>
+<DocsStory
+  title="Utseende"
```

Comment:
Tror kanskje jeg ville delt opp den i variant og størrelse.

Reply:
Med inspirasjon fra Aksel så syns jeg at størrelse alene i dokumentasjonen ikke har så mye for seg - trenger ikke ytterligere forklaring enn propsen og det som blir nevnt der tenker jeg🤓